test/mbta_v3_api/route_patterns/repo_test.exs

```diff
@@ -93,4 +94,58 @@ defmodule MBTAV3API.RoutePatterns.RepoTest do
       assert polyline
     end
   end
+
+  describe "included_stops_by_route_id/2" do
+    test "parses included stops out of route_patterns response" do
+      route_id = "Red"
+      response = File.read!(fixture_path("route_pattern_api_response.json")) |> JsonApi.parse()
```

Comment:
suggestion: Sorry, this is kind of an annoying request because there's nothing wrong with what you have here, but for consistency with how test data works in other parts of this library, could you please move the test data into an [ExMachina](https://hexdocs.pm/ex_machina/readme.html) factory? Example: [data](https://github.com/mbta/mbta_v3_api/blob/mss-more-files/test/support/factory.ex#L408) and [usage](https://github.com/mbta/mbta_v3_api/blob/mss-more-files/test/mbta_v3_api/schedules/repo_test.exs#L22). (Let me know if I can help clarify anything about how this works.)

Reply:
No problem